src/three/loaders/I3DMLoader.js

```diff
@@ -110,6 +110,14 @@ export class I3DMLoader extends I3DMLoaderBase {
 						const instances = [];
 						const meshes = [];
 						model.scene.updateMatrixWorld();
+						if ( rtcCenter ) {
+
+							model.scene.position.x += rtcCenter[ 0 ];
+							model.scene.position.y += rtcCenter[ 1 ];
+							model.scene.position.z += rtcCenter[ 2 ];
+
+						}
```

Comment:
Lets add this to each `InstancedMesh` in the traverse loop below, instead, so the scene object is set to the origin as every other model result is set to:

```js
instancedMesh.position.copy( averageVector );

if ( RTC_CENTER ) {

  instancedMesh.position.x += RTC_CENTER[ 0 ];
  // ...

}
```